pkgs/by-name/ka/kardolus-chatgpt-cli/package.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  makeWrapper,
+}:
+
+buildGoModule rec {
```

```suggestion
buildGoModule (finalAttrs: {
```
Go builder now supports `finalAttrs`-style lambdas, prefer them over rec whenever possible